tox.ini

```diff
@@ -111,6 +111,7 @@ deps =
     boto3
     juju==3.5.2.0
     git+https://github.com/canonical/saml-test-idp.git
+    websockets<14.0 # https://github.com/juju/python-libjuju/issues/1184
```

Comment:
It looks like it might no longer be needed :)

Reply:
Thanks, removed